be/src/exprs/aggregate_functions.cpp

```diff
@@ -231,6 +232,81 @@ void AggregateFunctions::avg_update(FunctionContext* ctx, const T& src, StringVa
     ++avg->count;
 }
 
+struct PercentileApproxState {
+public:
+    PercentileApproxState() {
```

Comment:
use Initializer list
```suggestion
    PercentileApproxState() : digest(new TDigest()) {
```